### ISSUE TITLE
Add support for Ghost

### ISSUE THREAD

**Issue by rebornix** [OpenLiveWriter/OpenLiveWriter#99]
Ghost is developed by part of original Wordpress developers and it's officially supported in Azure ( Azure Application ), we may want to put it in our blog providers list.

**Comment by Phasiq** [comment]
Yes, yes yes! Ghost is a fantastic blogging platform that certainly deserves support in Open Live Writer.

**Comment by rebornix** [comment]
@pekiZG is working on JsonBlogClient, if anyone is interested in Ghost support, you can sync with him.

**Comment by rebornix** [comment]
FYI. Ghost requires OAuth 2.0 for authentication, which is not supported perfectly in OLW for now, but we learn from https://github.com/OpenLiveWriter/OpenLiveWriter/pull/101

**Comment by pekiZG** [comment]
I'm here opened to colaboration;
Since I'm doing JsonBlogClient it would be awesome if there is someone willing to test stuff who is more familiar with Ghost.

**Comment by ygorlazaro** [comment]
I am able to test, I use Ghost a lot, hosted in Azure Web Sites

**Comment by swigerb** [comment]
I would like to assist and test Ghost support as well. I have a Ghost blog running on Azure (http://blog.mightybs.com/) and would like to see OLW work with Ghost. OAuth 2.0 implementation for Ghost should be straightforward.

**Comment by hashhar** [comment]
I recently migrated to Ghost too and have enough time on my hands now to be able to work on this. But I may need some mentoring and a quick overview of how the other blog providers work (the ones most similar to Ghost). @pekiZG @rebornix.

**Comment by szul** [comment]
Definitely would love to see support for Ghost. Has there been any movement on this?

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
